web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -435,6 +440,8 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
         }
       }
 
+      // Fix:  if flicks enabled, no roaming.
```

Comment:
Not sure what this comment means?

Reply:
```suggestion
```

It was likely a self-note that I forgot to remove once I accomplished the task.  I tend to leave those at end-of-day so that I don't forget loose ends.

L457's if-block was the 'fix', so it's safe to remove.